internal/porter_app/revisions.go

```
@@ -110,8 +110,10 @@ func EncodedRevisionFromProto(ctx context.Context, appRevision *porterv1.AppRevi
 	b64 := base64.StdEncoding.EncodeToString(encoded)
 
 	status, err := appRevisionStatusFromProto(appRevision.Status)
+	telemetry.WithAttributes(span, telemetry.AttributeKV{Key: "status", Value: string(status)})
 	if err != nil {
-		return revision, telemetry.Error(ctx, span, err, "error getting app revision status from proto")
+		// ignore error so that we can still return the revision
+		telemetry.WithAttributes(span, telemetry.AttributeKV{Key: "get-status-error", Value: err.Error()})
 	}
```

I would not swallow this error, and instead either do 
```
telemetry.WithAttributes(span, telemetry.AttributeKV{Key: "re-visionstatus", Value: string(appRevision.Status)})
_ = telemetry.Error(ctx, span, nil, "unknown revision type") // flagged as an error for visibilityu
```
or just don't flag it as an error and have a CCP notification on it

---

just don't want to normalize using values (like `status` here) if the error is not nil